src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+    private final int lengthNumber = 100;
```

Comment:
make it constant
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names